Client/mods/deathmatch/logic/CPacketHandler.cpp

```diff
@@ -626,28 +626,50 @@ void CPacketHandler::Packet_ServerDisconnected(NetBitStreamInterface& bitStream)
 
 void CPacketHandler::Packet_PlayerList(NetBitStreamInterface& bitStream)
 {
-    // bool                  - show the "X has joined the game" messages?
+        // bool                                 - show the "X has joined the game" messages?
     // [ following repeats <number of players joined> times ]
-    // unsigned char  (1)    - assigned player id
-    // unsigned char  (1)    - player nick length
-    // unsigned char  (X)    - player nick (X = player nick length)
-    // unsigned char  (1)    - nametag text length
-    // unsigned char  (X)    - nametag text (X = nametag text length)
-    // unsigned char  (3)    - nametag color
-    // unsigned char  (1)    - nametag showing
-    // bool                  - is he dead?
-    // bool                  - spawned? (following data only if this is TRUE)
-    // unsigned char  (1)    - model id
-    // ElementID      (2)    - team id
-    // bool                  - in a vehicle?
-    // ElementID      (2)    - vehicle id (if vehicle)
-    // unsigned char  (1)    - vehicle seat (if vehicle)
-    // CVector        (12)   - position (if player)
-    // float          (4)    - rotation (if player)
-    // bool                  - has a jetpack?
-    // unsigned short (2)    - dimension
-    // unsigned char  (1)    - fighting style
-    // Dear God, please fix issue #7376
+    // ElementID             (4)            - assigned player id(I'm not sure it's 4 bytes, but -- Pirulax)
```

Comment:
this could be releted with 4 byte address in memory? because tostring(element) return 8 numbers in hex